migration-util/src/main/java/edu/unc/lib/dcr/migration/premis/ContentPremisToRdfTransformer.java

```diff
@@ -83,8 +84,8 @@ public void compute() {
                 transformValidationEvent(eventEl);
             } else if (eventTypes.contains(INGESTION_TYPE)) {
                 addIngestionEvent(eventEl);
-            } else if (eventTypes.contains(NORMALIZATION_TYPE)) {
```

Comment:
This defines what we are migrating from, and we are migrating from boxc3 data which currently uses the Normalization event, so you'll need to leave it as NORMALIZATION in this case